web/modules/custom/hello_user/src/Controller/HelloUser.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Drupal\hello_user\Controller;
+use Drupal\Core\Controller\ControllerBase;
+
+/**
+ * Class HelloUser
+ * extends ControllerBase and contains functionality for module.
+ */
+class HelloUser extends ControllerBase {
+  
+  /**
+   *  Function view to render content with account name with hello-user theme.
+   * 
+   *  @return array with $content.
+   */
+  public function view() {
+    $content = [];
+    $content['name'] = \Drupal::currentUser()->getAccountName();
```

Comment:
This is not the proper way to call services in a Controller.
Please check the topic of Services and Dependency injection in Drupal docs.